src/components/pollutedLocationCard/PollutedLocationCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { BsTrashFill } from "react-icons/bs";
+import { BsTrash } from "react-icons/bs";
+import { RiLogoutCircleRFill } from "react-icons/ri";
+import PollutedLocationDTO from "../../types/backEnd/PollutedLocationDTO";
+
+type PollutedLocation = PollutedLocationDTO & {
+  title?: string;
+};
+
+const PollutedLocationCard: React.FC<PollutedLocation> = ({
```

Comment:
There's already a `PollutedLocation` type we defined earlier [here](https://github.com/vu-vibedosa/apsitvarkom-fe/blob/dev/src/types/PollutedLocation.ts) (#8). Use that (by importing it) instead of the DTO type